lib/server.js

```diff
@@ -129,13 +131,32 @@ module.exports = function createServer(config, log) {
     return P.all(promises)
   }
 
-  function allowWhitelisted (result, ip, email) {
-    // Regardless of any preceding checks, there are some IPs and emails
-    // that we won't block. These are typically for Mozilla QA purposes.
-    // They should be checked after everything else so as not to pay the
-    // overhead of checking the many requests that are *not* QA-related.
+  function isIpAllowed(ip) {
```

Comment:
With the introduction of these, wonder if we should we pull configs and logic into a common helper class and export `allowWhitelisted`?

Reply:
Does `isAllowed` on `allowedEmailDomains` and `allowedPhoneNumbers` fit that?